src/daemon/pcm.h

```diff
@@ -23,8 +23,8 @@
 	#include <signal.h>
 	#include <sys/time.h> // for gettimeofday()
 #endif
-#include "../../cpucounters.h"
-#include "../../utils.h"
+#include "../cpucounters.h"
+#include "../utils.h"
```

Comment:
This can drop ../ if the directory is already in the includes fro include_directories()